blueocean-github-pipeline/pom.xml

```diff
@@ -29,13 +29,13 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>github-organization-folder</artifactId>
-      <version>1.5</version>
+      <version>1.6</version>
```

Comment:
```
<optional>true</optional>
```

As this dependency is no longer required, but if user has it it must be upgraded